Tests/TestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace JakubZapletal\Component\BankStatement\Tests;
+
+
+class TestCase extends \PHPUnit_Framework_TestCase
+{
+}
+
+/*
+ * Convers E_RECOVERABLE_ERROR to exception.
+ * - used for tests relying on type hinting exceptions
+ */
+set_error_handler(function($errno, $errstr, $errfile, $errline) {
```

Comment:
Strange, I have tried to run tests on Docker image [PHP 5.6.6-cli](https://registry.hub.docker.com/_/php/) but everything is passing.